.github/workflows/go.yml

```diff
@@ -89,6 +89,7 @@ jobs:
       run: |
         echo "TEST_KUBECTL:$TEST_KUBECTL TEST_MINIKUBE_ENABLED:$TEST_MINIKUBE_ENABLED TEST_K8S_STATE: $TEST_K8S_STATE TEST_CONFIGMAP: $TEST_CONFIGMAP"
         export REPOPATH="${{ github.workspace }}"
+        export DOCKER_API_VERSION=1.39
```

Comment:
Do we need this environment variable in symphony-api containers?

Reply:
discussed offline. We don't want to set the environment variable in API container.